test/network_proposal_test.rb

```diff
@@ -20,27 +20,116 @@
   end
 
   describe "#make_proposal" do
-    it "returns a map with 'label_proposal' as an array with one line summary'" do
-      expect(subject.make_proposal({})["label_proposal"]).to eql(["one_line_summary"])
+    let(:using_wicked) { true }
+    let(:proposal) { subject.make_proposal({}) }
+
+    before do
+      allow(Yast::NetworkService).to receive(:wicked?).and_return(using_wicked)
+    end
+
+    it "returns a hash describing the proposal" do
+      expect(proposal).to include("label_proposal", "preformatted_proposal", "links")
     end
 
-    it "returns a map with 'preformatted_proposal' as an array with the network summary'" do
-      expect(subject.make_proposal({})["preformatted_proposal"]).to eql("rich_text_summary")
+    context "when using a Wicked backend" do
```

Comment:
NP: it is not `a` wicked backend but the `wicked` backend, same below with NetworkManager